components/src/core/info-list-item/info-list-item.tsx

```diff
@@ -150,6 +150,9 @@ export type InfoListItemProps = ViewProps & {
     /** Specifies whether to show color background around the icon */
     avatar?: boolean;
 
+    //** Custom icon to render to the left of the title */
+    customIcon?: JSX.Element;
+
```

Comment:
Instead of adding a new prop for this just to handle the menu header, let's use the leftComponent for now until we figure out how we want to refactor our approach to icons:
e.g.,
```
<InfoListItem leftComponent={<Avatar.Text size={40} label={'JB'} style={{marginLeft: 16}} />} />
```